util/dasharo/tag-with-changelog

```diff
@@ -200,7 +200,7 @@ for module in $(git submodule --quiet foreach pwd); do
 	declare "$version"="$(get_latest_commit_id "$module")"
 done
 
-git checkout origin/main  > /dev/null 2>&1
+git checkout $NEW_GIT_VERSION  > /dev/null 2>&1
```

Comment:
And well as this can be quoted.  This makes `shellcheck` happy (there are 2 stylistic notes unrelated to the changes).